lib/utils.js

```diff
@@ -0,0 +1,9 @@
+const publishEvent = (event, msg = {}) => {
```

Comment:
These functions are not needed, since it's just a renaming of standard functions